BUILD

```diff
@@ -0,0 +1,93 @@
+Building eudev
+===================
+
+The following instructions provide the steps needed to acquire and build the eudev
+software on the Debian, Devuan, and compatible Linux distributions.
+
+Please note that command examples in this document use the following convention. Commands
+prefixed by a "$" symbol are to be run as a regular, user. Commands with a "#" prefix
+need to be run as the root user. If your user has sudo access then sudo may be
+used instead of logging into the root account.
+
+
+1. Installing dependencies
```

Comment:
This is distro specific to Debian or Debian derivatives.  eudev should aim to remain distro agnostic.  You can either mark this section as "for Debian only" or you can write separate READMEs for each distro.

Reply:
Yes, that is why it specifically says the instructions are for Debian/Devuan at the top of the file. People are welcome to write instructions for their own distributions and add to the file.